src/Sulu/Bundle/ContentBundle/EventListener/WebspaceSerializeEventSubscriber.php

```diff
@@ -69,5 +69,13 @@ public function onPostSerialize(ObjectEvent $event)
 
         $portalInformation = $event->getContext()->accept(array_values($portalInformation));
         $visitor->addData('portalInformation', $portalInformation);
+
+        $urls = [];
+        foreach ($webspace->getPortals() as $portal) {
+            $environment = $portal->getEnvironment($this->environment);
+            $urls = array_merge($urls, $environment->getUrls());
+        }
+        $urls = $event->getContext()->accept($urls);
+        $visitor->addData('urls', $urls);
```

Comment:
its not really possible because we need the environment here. to achieve that we would have to inject the environment into the webspace. should i do that?